nodejs/awsx/ecs/cluster.ts

```diff
@@ -179,8 +179,10 @@ export interface ClusterArgs {
     vpc?: x.ec2.Vpc;
 
     /**
-     * An existing Cluster (or Cluster-Id) to use for this awsx Cluster.  If not provided, a default
-     * one will be created.
+     * An existing aws.ecs.Cluster (or the name of an existing aws.ecs.Cluster) to use for this
+     * awsx.ecs.Cluster.  If not provided, a default one will be created.
+     *
+     * Note: if passing a string, it must be the *name* of an existing Cluster, not the *id* of one.
```

Comment:
```suggestion
     * Note: If passing a string, use the *name* of an existing ECS Cluster instead of its *id*.
```

Wasn't sure if we wanted to be consistent with using `aws.ecs.Cluster` in the note but this should make it a little clearer.

Reply:
thanks!